test/setup/iam.tf

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  suffix = random_string.main.result
+  int_required_roles = [
+    "roles/owner"
+  ]
+
+  forseti_org_required_roles = [
+    "roles/resourcemanager.organizationAdmin",
+    "roles/compute.xpnAdmin",
+    "roles/iam.organizationRoleAdmin", // Permissions to manage/test real-time-enforcer roles
+    "roles/logging.configWriter",      // Permissions to create stackdriver log exports
+    "roles/bigquery.dataViewer",
+  ]
+
+  forseti_host_project_required_roles = [
+    "roles/owner",
+    "roles/compute.admin",
+    "roles/compute.securityAdmin",
+    "roles/billing.projectManager",
+    "roles/compute.networkAdmin",
+    "roles/compute.networkViewer",
+    "roles/compute.instanceAdmin",
+    "roles/iam.serviceAccountCreator",
+    "roles/iam.serviceAccountAdmin",
+    "roles/iam.serviceAccountUser",
+    "roles/serviceusage.serviceUsageAdmin",
+    "roles/storage.admin",
+    "roles/cloudsql.admin",
+    "roles/pubsub.admin",
+    "roles/bigquery.dataViewer",
+  ]
+
+  forseti_project_required_roles = [
+    "roles/owner",
```

Comment:
why do we need "roles/owner" as long it include all stuff like compute admin etc?

Reply:
I need it for role administration